### ISSUE TITLE
Visual Studio Code Extension - "Loading" with no errors

### ISSUE THREAD

**Issue by ProVega** [apache/camel-karavan#776]
Fresh install of Visual Studio Code, fresh install of the the latest extension.

Click Create Integration. It just spins at "Loading"

![image](https://github.com/apache/camel-karavan/assets/1735840/89e89700-696e-4f48-80a7-870fa18b708d)

How do I even troubleshoot this? Is there a log somewhere?

**Comment by mgubaidullin** [comment]
Look similar to this one 
https://camel.zulipchat.com/#narrow/stream/322293-camel-karavan/topic/.E2.9C.94.20VS.20code.20Karavan.20extension

**Comment by awesomeasaurus** [comment]
I had a similar issue in Windows which I solved by installing jbang using powershell and then assigning the newly created .jbang/bin folder in my PATH environment variables.

These powershell lines came from the [Camel Getting Started from CLI directions](https://camel.apache.org/manual/getting-started.html):
	iex "& { $(iwr https://ps.jbang.dev) } trust add https://github.com/apache/"
	iex "& { $(iwr https://ps.jbang.dev) } app install --fresh --force camel@apache/camel"

I believe I also needed to set Developer Mode to On in Windows Settings.

**Comment by jlepropre** [comment]
Hello,

I have found the solution.

As mentioned in https://camel.zulipchat.com/#narrow/stream/322293-camel-karavan/topic/karavan.20in.20VC.20on.20windows/near/299674615, it is first needed to select a folder.

**Comment by koszta5** [comment]
this is quite a silly limitation and should be more clearly reported

**Comment by mgubaidullin** [comment]
is it still a problem with 4.7.0?

**Comment by davsclaus** [comment]
Yes if you do "close folder" in the file menu, and then "create integration" then it keeps spinning
